frontend/src/components/CollectiveCard.js

```diff
@@ -29,11 +48,11 @@ export default class CollectiveCard extends Component {
             </div>
             <div className='CollectiveCard-body'>
               <div className='CollectiveCard-name'>{name}</div>
-              <div className='CollectiveCard-description'>{description}</div>
+              <div className='CollectiveCard-description'>We are on a mission to {description}</div>
```

Comment:
same language is too restrictive right now. Let's come back to it when we have a few hundred active collectives.